src/index.tsx

```diff
@@ -57,18 +51,20 @@ export const umamiAnalyticsContextFactory = function <
         >
           {children}
         </UmamiAnalyticsContext.Provider>
-        <Script
-          async
-          data-auto-track={autoTrack.toString()}
-          data-website-id={websiteId}
-          src={src}
-          {...(options.domains !== undefined &&
-            Array.isArray(options.domains) &&
-            options.domains.length > 0 && {
-              "data-domains": options.domains.join(","),
-            })}
-          onLoad={() => setLoaded(true)}
-        />
+        {!!src && !!websiteId && (
```

Comment:
Shouldn't return `null` instead of `false` to prevent rendering?

Reply:
Both null and false do not render anything.